src/test/scala/com/microsoft/hyperspace/index/IndexLogEntryTest.scala

```diff
@@ -350,8 +350,8 @@ class IndexLogEntryTest extends SparkFunSuite with SQLHelper with BeforeAndAfter
   test("Directory.fromDirectory and fromLeafFileswhere files are at same level but different" +
     "dirs.") {
     // File Structure
-    // temp/a/f1
-    // temp/b/f2
+    // testDir/temp/a/f1
+    // testDir/temp/b/f2
```

Comment:
This change is an resolution of [this comment](https://github.com/microsoft/hyperspace/pull/162/files#r489593365). It's repeated across many other tests.